CompetitionTemplate/src/competition/subsystems/vision/NetworkedJetsonServer.java

```diff
@@ -0,0 +1,102 @@
+package competition.subsystems.vision;
+
+import java.io.DataInputStream;
+import java.io.EOFException;
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.function.Consumer;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class NetworkedJetsonServer extends Thread implements JetsonServer {
```

Comment:
I'm not intending for this to be an `@Singleton` -- but the `JetsonServerManager` is.